make_mock_json.py

```diff
@@ -0,0 +1,413 @@
+        os.remove(user_list_path)
+
+    def create_a_mock_build(self, func, build_name, out_tag, build_args):
+        try:
+            build = func(**build_args)
+            build_id = build.get_build_name()
+            build = self.osbs.wait_for_build_to_get_scheduled(build_id)
+            self.osbs.watch_builds()
+            build = self.osbs.wait_for_build_to_finish(build_id)
+            self.osbs.get_build_logs(build_id)
+        except (subprocess.CalledProcessError, OsbsException):
+            pass
+
+        watch_data = canonize_data(copy.deepcopy(build.json), build_name, "Complete")
+        watch_obj = {
+            "object": watch_data,
+            "type": "MODIFIED"
+        }
+        out_name = "watch_build_test-" + out_tag + "build-123.json"
+        self.comp_write(out_name, watch_obj)
+
+        build_fname = "get-namespaces_osbs-stage_builds_" + build_id + "_-000.json"
+        build_path = "/".join([self.capture_dir, build_fname])
+        with open(build_path, "r") as infile:
+            build_data = canonize_data(json.load(infile), build_name, "Complete")
+            out_name = "build_test-" + out_tag + "build-123.json"
+            self.comp_write(out_name, build_data)
+        os.remove(build_path)
+        return out_name
+
+    def create_mock_build(self):
+        build_kwargs = {
+            'git_uri': self.osbs.build_conf.get_git_uri(),
+            'git_ref': self.osbs.build_conf.get_git_ref(),
+            'git_branch': self.osbs.build_conf.get_git_branch(),
+            'user': self.osbs.build_conf.get_user(),
+            'release': TEST_BUILD,
+            'platform': "x86_64",
+            'arrangement_version': DEFAULT_ARRANGEMENT_VERSION,
+            'scratch': True,
+        }
+
+        build_name = self.create_a_mock_build(self.osbs.create_worker_build, TEST_BUILD, "",
+                                              build_kwargs)
+        build_path = "/".join([self.mock_dir, build_name])
+        if os.stat(build_path):
+            created_path = "/".join([self.mock_dir,
+                                    "created_build_config_test-build-config-123.json"])
+            # build_test-build-config-123.json and created_build_config... are identical
+            # and os doesn't have a cp command, so create a link
+            try:
+                os.remove(created_path)
+            except:
+                pass
+
+            os.link(build_path, created_path)
+
+        del build_kwargs['platform']
+
+        self.create_a_mock_build(self.osbs.create_orchestrator_build, TEST_ORCHESTRATOR_BUILD,
+                                 "orchestrator-", build_kwargs)
+
+    def create_mock_imagestream(self):
+        print("look in {}".format(self.capture_dir))
+        #imagestream_repo =  "quay.io/prometheus/prometheus"
+        imagestream_server = "registry-proxy-stage.engineering.redhat.com"
+        imagestream_file = "/prometheus/prometheus"
```

Comment:
Let's take these (and the 3 tags) from command line parameters instead of hard-coding them.

Reply:
Fixed.